pkg/servers/servers.go

```diff
@@ -14,9 +14,10 @@ type ServerInfo struct {
 	HTTPSKeyFile    string
 	HTTPSCertFile   string
 	HasBothHandlers bool
+	FastShutdown    bool
 }
 
-func New(httpPort, httpsPort, HTTPSKeyFile, HTTPSCertFile string) *ServerInfo {
+func New(httpPort, httpsPort, HTTPSKeyFile, HTTPSCertFile string, fastShutdown bool) *ServerInfo {
```

Comment:
Changing this will require a change in the assisted service when we pull in the new code.

If we're only adding `fastShutdown` for tests I think it's probably better to not change this function signature and set the `FastShutdown` struct field directly in the tests.

Reply:
+1 to that, we don't want this to be configurable in production deployment